docs/guides/notebooks/configuration/groups.rst

```diff
@@ -0,0 +1,119 @@
+###################
+User-managed groups
+###################
+
+Create and manage closed user groups in order to share private files.
+
+.. _user-group-create:
+
+Create a group
+==============
+
+Groups are created an managed in the Comanage system at `id.lsst.cloud <https://id.lsst.cloud>`_
+(not from the terminal command line, as in some other systems).
+
+From the terminal command line it is possible to list all groups with ``getent group``.
+
+How to create a group in Comanage:
+
+* In a browser, navigate to `id.lsst.cloud <https://id.lsst.cloud>`_ and log in.
+
+* In the left menu sidebar, click on "Groups" and then "My Groups".
+
+* At right, a list of all the groups associated with your account will appear.
+
+* In the top horizontal menu, click on "Add Group".
+
+* Set the group properties:
+
+  * The group name should be short and must start with ``g_``.
```

Comment:
Oh we don't enforce that? Remind me to look at that

Reply:
Yeah if the user tries to define a group that doesn't start with ``g_`` they're denied, with a little red popup that says "Name must start with g_ and use only a-z,0-9,.,_, and -".